src/firefly/java/edu/caltech/ipac/firefly/server/servlets/AnyFileDownload.java

```diff
@@ -35,15 +36,32 @@
     public static final String LOG_PARAM   = "log";
     public static final String TRACK_PARAM = "track";
     public static final String USE_SERVER_NAME = "USE_SERVER_NAME";
+    public static final String FILE_LOCATION = "fileLocation";
+    public static final String WS_CMD = "wsCmd";
 
 
     protected void processRequest(HttpServletRequest req, HttpServletResponse res) throws Exception {
         SrvParam sp= new SrvParam(req.getParameterMap());
         String fname= sp.getRequired(FILE_PARAM);
         String local= sp.getOptional(RETURN_PARAM);
+        //String fileLocation = sp.getOptional(FILE_LOCATION);
         boolean log= sp.getOptionalBoolean(LOG_PARAM,false);
         boolean track= sp.getOptionalBoolean(TRACK_PARAM,false);
-
+/*
+        if (fileLocation != null && fileLocation.equals("isWs")) {
```

Comment:
Should go ahead and rollback changes to this file since all of the changes were commented out.

Reply:
Sorry, don't do anything here.  I took a closer look and it's best to leave it as is.  It may need to be refractored later.